src/mem/ruby/structures/RubyCache.py

```diff
@@ -44,6 +44,10 @@ class RubyCache(SimObject):
         "0B", "block size in bytes. 0 means default RubyBlockSize"
     )
 
+    # Atomic parameters only applicable to GPU atomics
+    atomicLatency = Param.Cycles(0, "Cycles for an atomic ALU operation")
```

Comment:
What does 0 latency mean, and why is that the default?

Reply:
This may be what the behavior was prior to the patch (assume no latency / latency is part of the enqueue latency for the slicc action)

Reply:
Ok. That information should be in a comment, then. Users will want to know what "0" means.

Reply:
Added a comment in the latest patchset.

Reply:
Thanks!